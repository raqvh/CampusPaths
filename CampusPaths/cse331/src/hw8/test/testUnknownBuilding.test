# Verify that the program finds and correctly outputs the format for
# unknown building
r
CSE
OOG

# Verify that the program finds and correctly outputs the format for
# two unknown buildings
r
OOG
GEE

# All test scripts should end with q to terminate the program.
q