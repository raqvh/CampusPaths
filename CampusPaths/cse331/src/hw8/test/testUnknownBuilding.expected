Menu:
	r to find a route
	b to see a list of all buildings
	q to quit

Enter an option ('m' to see the menu): # Verify that the program finds and correctly outputs the format for
# unknown building
Abbreviated name of starting building: Abbreviated name of ending building: Unknown building: OOG

Enter an option ('m' to see the menu): 
# Verify that the program finds and correctly outputs the format for
# two unknown buildings
Abbreviated name of starting building: Abbreviated name of ending building: Unknown building: OOG
Unknown building: GEE

Enter an option ('m' to see the menu): 
# All test scripts should end with q to terminate the program.
